content/en/blog/2024/llm-observability.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Observability for LLM-based applications using OpenTelemetry
```

Comment:
```suggestion
title: An Introduction to Observability for LLM-based applications using OpenTelemetry
```

Reply:
Rewording because this is a fairly deep topic, but the post contents are introductory in nature.